contents/docs/migrate/ingest-historic-data.mdx

```diff
@@ -4,7 +4,11 @@ sidebar: Docs
 showTitle: true
 ---
 
-> If you are planning on importing more than 20 million rows (or more than 10K requests per minute as part of your import), please contact us at [sales@posthog.com](mailto:sales@posthog.com) so we can make sure you are not rate limited.
+> Prior to beginning a historical data import please ensure that you've done the following:
+> 1. Create a project on our [US](https://us.posthog.com/signup) or [EU](https://eu.posthog.com/signup) Cloud
+> 2. Sign up to a paid plan on the billing page in-app (historic imports are not billable but this will unlock the necessary features).
```

Comment:
```suggestion
> 2. Sign [up to a paid plan on the billing page](https://us.posthog.com/organization/billing) (historic imports are not billable but this will unlock the necessary features).
```

Reply:
@joethreepwood this could be confusing if they have cloud accounts on US and EU, I see what you're doing as if there's only one account we should redirect but if there are two they could end up doing it in the wrong place.